thinkbox-deadline/azuredeploy.json

```diff
@@ -0,0 +1,388 @@
+                                "id": "[resourceId('Microsoft.Network/publicIPAddresses', concat('publicIp', copyindex()))]"
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "azure-repo",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts',parameters('storageAccount'))]",
+                "Microsoft.Network/networkInterfaces/nif",
+                "Microsoft.Compute/availabilitySets/availSet"
+            ],
+            "properties": {
+                "availabilitySet" :
+                {
+                    "id": "[resourceId('Microsoft.Compute/availabilitySets', 'availSet')]"
+                },
+                "hardwareProfile": {
+                    "vmSize": "Standard_A2"
+                },
+                "osProfile": {
+                    "computername": "azure-repo",
+                    "adminUsername": "[parameters('adminUserName')]",
+                    "adminPassword": "[parameters('adminPassword')]"
+                },
+                "storageProfile": {
+                    "osDisk": {
+                        "name": "osdisk",
+                        "osType": "Windows",
+                        "vhd": {
+                            "uri": "[concat('http://',parameters('storageAccount'),'.blob.core.windows.net/vhds/repo-osdisk.vhd')]"
+                        },
+                        "image":
+                        {
+                            "uri": "https://deadline.blob.core.windows.net/deadline72/azure-repo.vhd"
+                        },
+                        "caching": "ReadWrite",
+                        "createOption": "FromImage"
+                    }
+                },
+                "networkProfile": {
+                    "networkInterfaces": [
+                        {
+                            "id": "[resourceId('Microsoft.Network/networkInterfaces','nif')]"
+                        }
+                    ]
+                }
+            }
+        },
+
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "name": "azure-repo/autoconf",
+            "location": "[parameters('location')]",
+            "dependsOn": 
+            [
+                "Microsoft.Compute/virtualMachines/azure-repo"
+            ],
+            "properties":
+            {
+                "publisher": "Microsoft.Compute",
+                "type": "CustomScriptExtension",
+                "typeHandlerVersion": "1.4",
+                "settings":
+                {
+                    "fileUris": 
+                    [ 
+                        "https://deadline.blob.core.windows.net/scripts/repo-autoconf.py"

```

Comment:
Just checking that you want this source file at this location rather than checked into github somewhere